src/beast/beast/crypto/detail/sha_context.h

```diff
@@ -0,0 +1,322 @@
+//------------------------------------------------------------------------------
```

Comment:
Note: It's difficult to review this file. I downloaded the original file from from vog/sha1, and am attempting to match them up.